### ISSUE TITLE
NTR: Angiotensin-converting enzyme (ACE) high in blood (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8188]
New term request Angiotensin-converting enzyme (ACE) high in blood (segal_060926191459):   UMLS: 1867038

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000213
- label: Elevated circulating Angiotensin-converting enzyme concentration
- parents: [Abnormal circulating protein concentration (HP:0010876)](https://hpo.jax.org/app/browse/term/HP:0010876)
- pmid: 
- def: Concentration of angiotensin-converting enzyme (ACE) in the blood circulation above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8188
- synonyms: Angiotensin-converting enzyme (ACE) high in blood(NONE)
